build-tools/packages/build-tools/src/index.ts

```diff
@@ -14,6 +14,7 @@ export type { Logger } from "./common/logging";
 export { MonoRepo } from "./common/monoRepo";
 export {
 	Package,
+	type PackageDependency,
```

Comment:
Is there any way to not add to the exported surface of build-tools? Trying to limit build-tools exports since ultimately it needs to go away/become fluid-build only.

Reply:
Sure - the outside use doesn't currently make use of production/dev/peer knowledge anyway. Removed.
General workaround is to use the type of a member, in this case `combinedDependencies`, instead of the type, but that is not as straight-forward for `Generators`. Type separation is fine.